### ISSUE TITLE
Accessibility finding: Update contact information heading not descriptive

### ISSUE THREAD

**Issue by emisparkles** [department-of-veterans-affairs/va.gov-team#133935]
### Checklist item

WEB-242	Each page has a unique, descriptive title

### Severity level

4, Low. Consider fixing or exploring in 2-4 sprints post-launch.

### Page(s)/url(s)

Profile > Scheduling Preferences > Edit preferred contact method > Update contact information
- https://staging.va.gov/profile/edit?returnPath=%2Fprofile%2Fhealth-care-settings%2Fscheduling-preferences&fieldName=email
- https://staging.va.gov/profile/edit?returnPath=%2Fprofile%2Fhealth-care-settings%2Fscheduling-preferences&fieldName=homePhone
- https://staging.va.gov/profile/edit?returnPath=%2Fprofile%2Fhealth-care-settings%2Fscheduling-preferences&fieldName=mobilePhone
- https://staging.va.gov/profile/edit?returnPath=%2Fprofile%2Fhealth-care-settings%2Fscheduling-preferences&fieldName=workPhone
- https://staging.va.gov/profile/edit?returnPath=%2Fprofile%2Fhealth-care-settings%2Fscheduling-preferences&fieldName=mailingAddress

### Details

When updating contact information through the "Edit contact info as a subtask flow" from the scheduling preferences page, the page title says "Edit contact preferences". This heading is not specific to this part of the edit flow.

<img width="373" height="484" alt="Image" src="https://github.com/user-attachments/assets/a1c5cb34-f507-405b-be88-32fca00c6e27" />

### Reproduction steps

1. Log into staging.va.gov
2. Navigate to profile/scheduling-preferences
2. Under Contact preferences section, click Edit for "What's the best way to contact you to schedule your appointments?"
3. Note page title is "Edit contact preferences"
4. Select a contact method (phone, email, mailing address) and click Continue
5. Click Update information
6. Note page title remains "Edit contact preferences"

### Proposed solution or next steps

Update page title at this step of the flow to say "Update your [contact information]" following the pattern from the notification settings page (found in [Edit.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/7a3622cdee9b2c86a33a38ef42732cc52cde92eb/src/applications/personalization/profile/components/edit/Edit.jsx#L122-L139)). Ensure this matches [current designs](https://www.figma.com/design/smldak4d56moGObUR5jifE/E.-Dole-Act--Appointment-Scheduling-Preferences-in-VA-Profile?node-id=842-59760&t=3ji3xBqQjjdpGFER-4).

Examples from Notifications settings:

<img width="373" height="416" alt="Image" src="https://github.com/user-attachments/assets/3571cefc-a9d4-46c8-8473-a36d29184869" />

<img width="371" height="414" alt="Image" src="https://github.com/user-attachments/assets/0c02452d-b1e6-4459-a7be-7afa3f28eb04" />

**Comment by alatempa** [comment]
@jfinnerty13 to make a UX task to update designs

**Comment by jfinnerty13** [comment]
@emisparkles can you capture what the header would look like for all the other options please? Including home address, mailing address, contact email, work phone, and home phone? Just want to make sure there's nothing weird in there. That was the big reason we decided not to move forward with this because it would generate strange, so want to confirm it won't.

Side note - can you ping me in the engineering channel too please next time? Just so others can see it too in case I can't respond. Thanks!

**Comment by emisparkles** [comment]
@jfinnerty13 Here are the other headers in the scheduling preferences flow if using the same logic as email and text notifications (home address isn't an option) - all can be Add or Update depending on whether that info is in the profile.

<img width="365" height="232" alt="Image" src="https://github.com/user-attachments/assets/70417ba7-01cf-4f37-8c36-b8620a87732c" />
<img width="366" height="238" alt="Image" src="https://github.com/user-attachments/assets/96cc101e-6f29-465c-9495-efa1a2124b94" />
<img width="369" height="234" alt="Image" src="https://github.com/user-attachments/assets/36696a9a-54da-429c-a76f-040dc09b212a" />
<img width="354" height="239" alt="Image" src="https://github.com/user-attachments/assets/1c0ab46b-5412-4ae8-837e-de53693ffda2" />

**Comment by emisparkles** [comment]
Confirmed in this [Slack thread](https://dsva.slack.com/archives/C08J1GB54VD/p1775825974591939) that we want the "[Add/Update] your [contact info type]" version using the same logic as the notifications subflow.

[Updated designs](https://www.figma.com/design/5bxmQiCLaXdXrEfPhqMmdJ/Profile-2.0--113546?node-id=2011-2020&t=kLIvaYJlRDl2GbvB-1)

**Comment by emisparkles** [comment]
Validated no regressions to email and text notifications edit-as-a-subtask flow in staging. We won't be able to validate the changes to scheduling preferences flows since we don't have any staging users in a pilot VISN, however these changes have been tested and validated locally.

**Comment by emisparkles** [comment]
@alatempa @jfinnerty13 I created a [followup ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/139718) to validate this work once we have access to the scheduling preferences page in staging.
